api/tests/test-router.js

```diff
@@ -78,6 +78,8 @@ o.spec("route", function() {
 					callAsync(function() {
 						o(root.firstChild.nodeName).equals("DIV")
 
+						o($window.location.pathname).equals("/a")
```

Comment:
location.pathname is not necessarily `/a`. This test happens 12 times for different routing prefix modes, so it could be e.g. `/#!/a` or `/foo/a`. This should be `o(route.get()).equals("/a")`